VirtoCommerce.Platform.Modules/Bundling/ScriptCollector.cs

```diff
@@ -0,0 +1,34 @@
+using System.Linq;
```

Comment:
we can remove it and use one BundleProvider for styles/scripts because we know file extension